manifests/create_bolt_workshop_master.pp

```diff
@@ -36,7 +36,6 @@
   awskit::create_host { $instance_name:
     ami             => $awskit::centos_ami,
     instance_type   => $instance_type,
-    user_data       => '',
```

Comment:
I had to specify something for user_data in the original setup; is user_data_template now truly optional?

Reply:
@kreeuwijk There are the define parameters now:
```
define awskit::create_host (
  $ami,
  $instance_type,
  $user_data          = undef,
  $user_data_template = undef,
  $master_ip          = undef,
  $master_name        = undef,
  $run_agent          = true,
  $security_groups    = 'none',
  $key_name           = 'none',
  $role               = undef,
  $environment        = undef,
  $public_ip          = undef,
){
```
Both `$user_data` and `$user_data_template` are optional now. To be more sure of the correct workings, you should add spec tests for your new bolt-workshop-related classes.